src/custom-ci-info-bot.ts

```diff
@@ -0,0 +1,31 @@
+import * as probot from 'probot';
+
+// https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab
+const dummy = `
+<details>
+<summary>Click to expand...</summary>
+
+_Hello,_ **world!**
+</details>
+`.trim();
+
+// this has other fields too but we don't use them
+interface Label {
+  name: string;
+}
+
+export default function myBot(app: probot.Application): void {
+  app.on('pull_request.edited', async context => {
+    const {owner, repo} = context.repo();
+    const {number, labels, body} = context.payload.pull_request;
+    context.log.info({owner, repo, number, body}, 'custom CI info start');
+    if (!labels.map((label: Label) => label.name).includes('ci/future')) {
```

Comment:
nit, the literal `ci/future` can be DRY extracted into a const variable to ease future changes

Reply:
good point, done